src/sentry/api/endpoints/organization_auth_providers.py

```diff
@@ -20,7 +22,15 @@ def get(self, request: Request, organization) -> Response:
         :auth: required
         """
         provider_list = []
+        print("org in list", organization)
```

Comment:
[2] but here the organization doesn't have subscription so the test fails with: 'Organization' object has no attribute 'subscription' if I print subscription